app/src/main/python/gogdl/cli.py

```
@@ -0,0 +1,177 @@
+def handle_auth(arguments, api_handler):
+    """Handle GOG authentication - exchange authorization code for access token or get existing credentials"""
+    logger = logging.getLogger("GOGDL-AUTH")
+    
+    try:
+        import requests
+        import os
+        import time
+        
+        # GOG OAuth constants
+        GOG_CLIENT_ID = "46899977096215655"
+        GOG_CLIENT_SECRET = "9d85c43b1482497dbbce61f6e4aa173a433796eeae2ca8c5f6129f2dc4de46d9"
```

Ruff (0.12.2)

Possible hardcoded password assigned to: "GOG_CLIENT_SECRET"

(S105)